editor/editor_about.cpp

```diff
@@ -72,17 +88,15 @@ ScrollContainer *EditorAbout::_populate_list(const String &p_name, const List<St
 			vbc->add_child(lbl);
 
 			ItemList *il = memnew(ItemList);
-			il->set_max_columns(16);
 			il->set_h_size_flags(Control::SIZE_EXPAND_FILL);
 			il->set_same_column_width(true);
 			il->set_auto_height(true);
+			il->set_mouse_filter(Control::MOUSE_FILTER_IGNORE);
 			while (*names_ptr) {
-				il->add_item(String::utf8(*names_ptr++), NULL, false);
+				il->add_item(" " + String::utf8(*names_ptr++) + " ", NULL, false);
```

Comment:
Is there no decent item padding feature in ItemList to avoid such a hack? :D

Reply:
I forgot about theme constants. :P `hseparation` might be useful here.